.github/workflows/sync_dashboard_crds.yaml

```diff
@@ -0,0 +1,29 @@
+# Opens a PR in the opendatahub-operator repo when crds are updated in the dashboard repo
+name: Sync Dashboard CRDs
+
+# Await dispatch from dashboard repo that crds have been modified
+on:
+  workflow_dispatch
+
+jobs:
+  dashboard-sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: '0'
+      - name: Gather files
+        shell: bash
+        run: |
+          cd ${{ github.workspace }}/config/crd/dashboard-crds
+          svn checkout --force https://github.com/opendatahub-io/odh-dashboard/branches/main/manifests/crd
+          svn revert -R ./crd
```

Comment:
@maroroman Not exactly sure on how the `svn revert` command working here? Is it reverting any uncommited changes?

Can we instead just remove the existing version of crds and replace them with `odh-dashboard/branches/main/manifests/crd` ?

Reply:
That is what the command should do, as far as I know, when I looked it up and tested it the revert command with the -R argument "reverts" the state of the folder to whatever has been checked out. So this basically overwrites whatever is in the crd folder with the crd changes in the dashboard.

Reply:
Okay I replaced the checkout and revert command with a better `svn export` command. I have verified that it works locally. Could you please retry your test @VaishnaviHire?